src/_posts/2020-05-18-building-a-business-case-for-your-software-modernisation-initiative.md

```diff
@@ -65,7 +65,7 @@ The main topics you should focus on:
 - What the payoff is for the audience and business
 
 
-On this particular topic, you may want to check out our <a href="https://info.codurance.com/en/case-for-change-questionnaire" target="_blank">list of questions to help you gather the information that you need as part of your research</a>.
+On this particular topic, you may want to check out our <a href="{% t software-modernisation.cheat-sheet-cta-href %}" target="_blank">list of questions to help you gather the information that you need as part of your research</a>.
```

Comment:
Maybe we can check with @nicholemellekas on this, but this seems like an awfully long piece of text for a link 😅